android/fastlane/Fastfile

```diff
@@ -0,0 +1,132 @@
+                if UI.confirm(text:"Are you sure you want to continue?")
+                    do_commit_and_push_tags(newVersion: newVersion)
+                else
+                    UI.error errorMessageCancelled
+                end
+            else
+                UI.error errorMessageCancelled
+            end
+        end
+
+    end
+
+    private_lane :do_commit_and_push_tags do |options|
+        newVersion = options[:newVersion]
+
+        set_version_number(newVersion: "#{newVersion}")
+
+        git_commit(
+            message: "Release #{newVersion}",
+            path: "*",
+            allow_nothing_to_commit: true,
+            skip_git_hooks: true
+        )
+
+        sh "git tag -a #{newVersion} -m '#{newVersion}'"
+        UI.header("#{newVersion} release successfully created. Commit info: #{last_git_commit}")
+
+        if UI.confirm(text:"Do you want to push the tags to the remote?")
+            push_to_git_remote(
+                remote: "origin",
+                tags: true
+            )
+        else
+            UI.error errorMessageCancelled
+        end
+
+        nextSnapshot = prompt(text: "\nEnter Next Snapshot Version: ")
+        if nextSnapshot == ""
+            UI.user_error!("Next Snapshot Version is required")
+        end
+
+        set_version_number(newVersion: "#{nextSnapshot}-SNAPSHOT")
+        git_commit(
+            message: "Prepare next development version.",
+            path: "*",
+            allow_nothing_to_commit: true,
+            skip_git_hooks: true
+        )
+        UI.header("#{newVersion}-SNAPSHOT version successfully set. Commit info: #{last_git_commit}")
+
+        if UI.confirm(text:"Do you want to push changes to remote?")
+            push_to_git_remote(
+                remote: "origin",
+                tags: true
+            )
+        else
+            UI.error errorMessageCancelled
+        end
+
+
+    end
+
+    desc "Prompt for version number"
+    private_lane :get_version_number do |options|
+        release_number = options[:release_number]
+        snapshot_version = get_version_from_properties
+        if release_number == nil && snapshot_version == nil
+            prompt(text: "\nLast release was: #{last_git_tag}\nEnter New Version Number:")
+        elsif snapshot_version != nil
+            UI.message("Snapshot version: #{snapshot_version}")
+            snapshot_version
+        else
+            release_number
+        end
+    end
+
+    desc "Update version number"
+    private_lane :set_version_number do |options|
+        newVersion = options[:newVersion]
+
+        File.open('../version.properties', 'w') do |file|
+            file.write("VERSION_NAME=#{newVersion}")
+        end
+    end
+
+    desc "Get version from version properties"
+    private_lane :get_version_from_properties do |options|
+        version_name = File.read('../version.properties').match(/VERSION_NAME=(.*)/)[1]
```

Comment:
Nit: in the main browser fastlane, we use a plugin which understands `.properties` files so we have less manual parsing to do. I see you already have it defined: `fastlane-plugin-property_file_read`

could you use that here to read the version with less manual parsing?

e.g., from browser: `version = property_file_read(file: "app/version/version.properties")["VERSION"]`

Reply:
Ah nice. Will do 🙏